File: tests/07_phases_omp.dg/common/fortran/failure_task_01.f90
! <testinfo>
! test_generator=config/mercurium-omp
! test_compile_fail=yes
! </testinfo>

PROGRAM P
    INTEGER :: A(10)

    !$OMP TASK FIRSTPRIVATE(A(3))
        CONTINUE
    !$OMP END TASK

    !$OMP TASKWAIT
END PROGRAM P

File: tests/07_phases_omp.dg/common/cxx/failure_task_01.cpp
/*
<testinfo>
test_generator=config/mercurium-omp
test_compile_fail=yes
</testinfo>
*/
struct A
{
    int m[4];
    void foo()
    {
        #pragma omp task firstprivate(m[2])
        {
        }
    }
};

int main()
{
   A a;
   a.foo();
#pragma omp taskwait
}
